index.android.js

```diff
@@ -14,7 +14,9 @@ const PepperoniAppTemplate = React.createClass({
 
   navigateBack() {
     const navigationState = store.getState().get('navigationState');
-    const currentTab = navigationState.getIn(['routes', navigationState.get('index')]);
+    const tabs = navigationState.get('tabs');
+    const tabKey = tabs.getIn(['routes', tabs.get('index')]).get('key');
+    const currentTab = navigationState.get(tabKey);
 
```

Comment:
changed the way to find `currentTab` accordingly to the `navigationState` changes